client/src/reducers/exerciseEvents.js

```diff
@@ -6,7 +6,7 @@ export default function exerciseEventsReducer(state = initialState.exerciseEvent
     case EXERCISE_EVENT_CREATED:
       return [...state, action.data]
     case USER_LOGIN_SUCCESS:
-      return action.data.exerciseEvents
+      return action.data.exerciseEvents || null
```

Comment:
~i think you need to return exerciseEvents in the update route as well~

do `return action.data.exerciseEvents || state` instead